webpack/assets/javascripts/bundle.js

```diff
@@ -8,6 +8,8 @@ require('expose?JsDiff!diff');
 require('./bundle_flot');
 require('./bundle_select2');
 require('./bundle_datatables');
+var Turbolinks = require('turbolinks');
```

Comment:
This code creates a global. This is a very bad practice for javascript. We don't allow that, except in the case Tomer mentioned below. If the expose loader doesn't work for you, please add Turbolinks as a property of the window.tfm object.